logdna/provider_test.go

```diff
@@ -8,6 +8,7 @@ import (
 )
 
 var serviceKey = os.Getenv("SERVICE_KEY")
+var apiHostUrl = os.Getenv("API_URL")
```

Comment:
+1 this is helpful and I had been meaning to do this. we could also consider adding a secondary set of credentials in CI and running non-`main` branches against staging